.github/workflows/new_stats_build.yaml

```diff
@@ -0,0 +1,45 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code using yarn build:app
+
+name: New Stats Build
+
+on:
+  push:
+    branches:
+      - development
+    paths:
+      - "packages/new_stats/**"
+  pull_request:
+    branches:
+      - development
+    paths:
+      - "packages/new_stats/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: "--max-old-space-size=8192"
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "yarn"
+          cache-dependency-path: "**/yarn.lock"
+
+      - name: Install dependencies
+        run: |
+          yarn install
+
+      - name: Build
+        run: |
+          lerna run build --no-private
+          yarn workspace @threefold/newstats build
```

Comment:
i believe it should be `newstats` only without `@threefold`

Reply:
as per @AhmedHanafy725 comment ,this prefix should be added from your side